level1/f256/modules/vtio.asm

```diff
@@ -1272,6 +1329,31 @@ s4@                 sta       R$A,u               store buttons in caller's A
                     clrb                          clear carry
                     rts                           return
 
+		    ifgt      Level-1
```

Comment:
Why is this conditional here if we support Level 1?

Reply:
Removed support for Level-1 to remove from FEU. Since FEU uses L1 vtio, it was easier to strip the mouse from L1 than to make a conditional for the FEU.